src/main/resources/persistence.properties

```diff
@@ -0,0 +1,17 @@
+hibernate.dialect = org.hibernate.dialect.PostgreSQL9Dialect
+hibernate.connection.driver_class = org.postgresql.Driver
+hibernate.archive.autodetection = class
+
+javax.persistence.jdbc.url = jdbc:postgresql://188.166.92.29:5432/moodcat_db
+javax.persistence.jdbc.user = moodcat
+javax.persistence.jdbc.password = INSERT_POSTGRES_PASSWORD
```

Comment:
According to [StackOverflow](http://stackoverflow.com/questions/936249/stop-tracking-and-ignore-changes-to-a-file-in-git) `git update-index --assume-unchanged src/main/resources/persistence.properties` works. This made the file not trackable and not removed from the git repository itself.